app/ui/design-system/src/lib/Components/InternalCodeblock/index.tsx

```diff
@@ -86,10 +86,10 @@ export function InternalCodeblock({
         <Code
           language={children.props.className}
           code={codeString}
-          innerClasses={clsx(
-            "w-full",
-            tall ? "max-h-[280px]" : "max-h-[130px]"
-          )}
+          innerClasses={clsx("w-full")}
+          innerStyle={{
+            maxHeight: constrained ? 280 : `calc(100vh - ${NAV_BARS_HEIGHT}px)`,
```

Comment:
I don't think this is _ideal_ because it assumes an `InternalCodeblock` is always going to be rendered in the context of a `NavigationBar` and `InternalSubnav`. Plus those heights could easily change, and they aren't really guaranteed as-is. But since we're already using that elsewhere it works for now.

Reply:
@joekrill the goal was to just maximize the code block height on taller windows, and I wanted to avoid querying the dom for the parent's height. The nav height is fixed using the same const, and there's a buffer to give it some extra space.

Good point about assuming this will always be rendered on internal pages, I changed the default state to a fixed height in #367

Thanks for reviewing!